languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -4042,6 +4042,65 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
             <example>Ces pu(ri)tains de Suisses ils ont peur de voir LES CANICULES S'EMBALLER !</example>
             <example>La république a dans les faits duré peu de temps.</example>
         </rule>
+        <rule id="D_N_J_VPPA" name="L'expérience journalistique engagé (engagée)" default="temp_off">
```

Comment:
Nice! We actually don't find the corrected example yet 🤓

I've found FPs in the rule editor:

> Étienne Davignon, à l'époque jeune **attaché** aux affaires étrangères.
> Le discours de bienvenue à la délégation soviétique **prononcé** par Gustáv Husák est parfaitement en ligne avec le contenu du traité.
> En 1844, il a publié un essai défendant la prosodie romantique intitulé Prosodie de l'école moderne, précédé d'une lettre de Victor Hugo et préfacé par Émile Deschamps.

Possible pitfalls are nouns that look like `V ppa.*` and more complex attribute structures (with `V ppr.*` for example).